dist/angular-patternfly.js

```diff
@@ -1978,6 +1990,176 @@ angular.module('patternfly.select', []).directive('pfSelect', function ($timeout
     }
   };
 });
+;
+/**
+ * @ngdoc directive
+ * @name patternfly.utils.directive:pfTransclude
+ * @restrict A
+ * @element ANY
+ * @param {string} pfTransclude specifies the type of transclusion to use.<br/>
+ * <strong>Values:</strong>
+ * <ul style='list-style-type: none'>
+ * <li> 'sibling' - The transcluded contents scope is a sibling one to the element where transclusion happens (default)
+ * <li> 'parent'  - The transcluded contents scope is that of the element where transclusion happens.
+ * <li> 'child'   - The transcluded contents scope is child scope to the scope of the element where transclusion happens.
+ * </ul>
+ *
+ * @description
+ *   Directive for transcluding in directives and setting up scope of children of parent directives. This is a workaround
+ *   for https://github.com/angular/angular.js/issues/5489
+ *
+ * @example
+<example module="patternfly.utils">
+  <file name="index.html">
+    <style>
+
+      .pf-transclude-example div {
+        border: 1px solid #337ab7;
+        margin-bottom: 20px;
+        margin-left:   20px;
+      }
+
+      .pf-transclude-example p {
+        background-color: #337ab7;
+        margin: 0;
+        padding: 5px 10px;
+      }
+
+     .pf-transclude-example id {
+        display: inline-block;
+        background-color: #def3ff;
+        color: #000000;
+        border-radius: 10px;
+        width: 20px;
+        height: 20px;
+        text-align: center;
+        line-height: 20px;
+        margin-left: 5px;
+      }
+
+      .pf-transclude-example pre {
+        padding: 5px;
+        border-width: 0px;
+      }
```

Comment:
Why not just create another `<file>` with the styles in it instead of putting them in the html?